components/lvgl_esp32_drivers/lvgl_tft/rm67162.c

```diff
@@ -85,7 +85,7 @@ void rm67162_init(void)
 		rm67162_send_cmd(rm67162_init_table[cnt].cmd);
 		rm67162_send_data((uint8_t*)rm67162_init_table[cnt].data, rm67162_init_table[cnt].databytes & 0x7F);
 		if (rm67162_init_table[cnt].databytes & 0x80) {
-			vTaskDelay(120 / portTICK_RATE_MS);//deafult 100ms
+			vTaskDelay(120 / portTICK_PERIOD_MS);//deafult 100ms
```

Comment:
you should not change any part of the code out of your task scope

Reply:
because my project did not compile !